drivers/vergesense/room_sensor.cr

```diff
@@ -62,9 +62,10 @@ class Vergesense::RoomSensor < PlaceOS::Driver
       self[:people] = 0
     end
 
+    iaq = floor_space.environment.try &.iaq
     self[:humidity] = floor_space.environment.try &.humidity.value
     self[:temperature] = floor_space.environment.try &.temperature.value
-    self[:air_quality] = floor_space.environment.try &.iaq.value
+    self[:air_quality] = iaq.nil? ? nil : iaq.not_nil!.value
```

Comment:
```suggestion
    self[:air_quality] = floor_space.environment.try(&.iaq.try(&.value))
```

Not ideal, but maybe it's worth allowing the other sensors to be nil, and creating a helper that can warn when an integration is not returning values. Suggestion below might be a bit overkill

```crystal
module Vergesense
  struct Environment
    {% begin %}
    {% for reading in [iaq, temperature, humidity] %}
      getter {{ reading }} : Sensor?
      
      def read_{{ reading }}
        if reading = {{ reading }}
          reading.value
        else
          Log.warn { "missing reading for {{ reading }}" }
          nil
        end
      end
    {% end %}
    {% end %}
  end
end
```